digital_land/command_arguments.py

```diff
@@ -35,6 +35,24 @@ def issue_dir(f):
     )(f)
 
 
+def performance_dir(f):
+    return click.option(
+        "--performance-dir",
+        "-i",
+        type=click.Path(exists=True),
+        default="performance/",
+    )(f)
+
+
+def operational_issue_dir(f):
```

Comment:
we're moving this to the performance directory so this default may cause it to break

Reply:
I've updated this so the default is performance/operational_issue